app/views/escorts/confirm_cancel.html.slim

```diff
@@ -0,0 +1,17 @@
+= content_for :breadcrumbs do
+  - breadcrumbs_for_page root: true do
+    - breadcrumb detainee_breadcrumb(escort.detainee), escort_path(escort)
+
+h1
+  = t('.cancelling_per_for')
```

Comment:
This [comment](https://github.com/ministryofjustice/moving-people-safely/pull/337#discussion_r125449193) I think is still relevant here. Why are we creating random locale names dependant on the actual locale translation? 

IMO, this is the most basic stuff we do on locales: they have a context, the context should be pretty clear when I'm looking at it in the locales files. This one, on the other hand is anything but clear. It's clear for me it's the title of the form, I'm just missing to understand why we don't keep the locales to this very clear namespaces...